src/core/mac/link_raw.cpp

```diff
@@ -58,6 +58,8 @@ LinkRaw::LinkRaw(Instance &aInstance)
     , mSubMac(aInstance.Get<SubMac>())
 #endif
 {
+    otPlatCryptoInit();
```

Comment:
Do we need this call to `otPlatCryptoInit()` here?

The `otPlatCryptoInit()` is already called from the `KeyManager` constructor.

I assume this was added here for the RCP build. However, `LinkRaw` can also be included in an FTD/MTD build, which would result in `otPlatCryptoInit()` being called twice.

A better approach can be to move the `otPlatCryptoInit()` call to the `Instance` constructor, ensuring it is called once for all build types (FTD/MTD or RCP)?